src/main/java/fi/okm/jod/yksilo/domain/PolunVaiheLuontiTyyppi.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2025 The Finnish Ministry of Education and Culture, The Finnish
+ * The Ministry of Economic Affairs and Employment, The Finnish National Agency of
+ * Education (Opetushallitus) and The Finnish Development and Administration centre
+ * for ELY Centres and TE Offices (KEHA).
+ *
+ * Licensed under the EUPL-1.2-or-later.
+ */
+
+package fi.okm.jod.yksilo.domain;
+
+public enum PolunVaiheLuontiTyyppi {
```

Comment:
LuontiTyyppi sounds a bit odd, perhaps source (lähde; PolunVaiheLahde) reads better; the source/origin of the vaihe is either an user or a suggestion)